File: app/src/main/java/com/example/appswim/database/SwimmerDatabase.java
package com.example.appswim.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.example.appswim.Swimmer;

@Database(entities = {Swimmer.class},version = 1)
public abstract class SwimmerDatabase extends RoomDatabase {
    public abstract SwimmerDao swimmerDao();
}

File: app/src/main/java/com/example/appswim/database/SwimmerDatabase.kt
package com.example.appswim.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

import com.example.appswim.models.Swimmer

@Database(entities = [Swimmer::class], version = 4)
abstract class SwimmerDatabase : RoomDatabase() {
    abstract val swimmerDao: SwimmerDao
}
